File: programmers/js/level01/12903.js
// 가운데 글자 가져오기
function solution(s) {
  const lenDiv2 = Math.floor(s.length / 2);
  
  return s.length % 2 === 0 ? 
    s.slice(lenDiv2 - 1, lenDiv2 + 1) : 
    s.at(lenDiv2);
}

File: programmers/java/level01/12903.java
// 가운데 글자 가져오기

class Solution {
    public String solution(String s) {
        return s.length() % 2 == 0 
            ? s.substring(s.length() / 2 - 1, s.length() / 2 + 1)
            : s.substring((int) Math.floor(s.length() / 2), 
                          (int) Math.floor(s.length() / 2) + 1);
    }
}